changelogs/fragments/lie_fix_plugin_hash_string_return.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - mysql_info - Add ``plugin_hash_string`` to ``users_info`` filter's output. The existing ``plugin_auth_string`` contained the hashed password and thus is missleading, it will be removed from community.mysql 4.0.0. (https://github.com/ansible-collections/community.mysql/pull/629).
```

Comment:
```suggestion
  - mysql_info - Add ``plugin_hash_string`` to ``users_info`` filter's output. The existing ``plugin_auth_string`` contained the hashed password and thus is missleading, it will be removed from community.mysql 4.0.0. (https://github.com/ansible-collections/community.mysql/pull/629).
  
 breaking_changes:
 - mysql_info - The existing ``plugin_auth_string`` return value contains the hashed password and it's missleading, it will be removed from community.mysql 4.0.0. Use the `plugin_hash_string` return value instead (https://github.com/ansible-collections/community.mysql/pull/629).
```